heron/config/src/yaml/conf/local/healthmgr.yaml

```diff
@@ -18,27 +18,32 @@
 # Topology health manager mode:
 # disabled = do not launch the health manager
 # cluster = launch the health manager on container-0
```

Comment:
1. Can we use `enable/disable` consistently?

2. How about the other example config files? Usually we want to keep them consistent

Reply:
1. The original mode includes disabled, cluster, standalone; rather than 2 dimensions.
2. updated the other config files